pkg/controller/servicemesh/webhooks/mutation/controlplane.go

```diff
@@ -169,6 +167,8 @@ type smcpmutator interface {
 	GetPatches() []jsonpatch.JsonPatchOperation
 	IsOpenShiftRouteEnabled() *bool
 	SetOpenShiftRouteEnabled(bool)
+	IsTracingTypeSpecified() bool
+	DisableTracing(v2.TracerType)
```

Comment:
As I said in the previous comment, this function always takes the same argument, so it can take no argument and just always set `v2.TracerTypeNone`.
```suggestion
	DisableTracing()
```

Reply:
Fixed this now